index.html

```diff
@@ -1,216 +1,283 @@
-                    <div class="form-group">
-                        <input class="input-text" type="email" id="email" name="email" placeholder="Email address"
-                            required>
-                    </div>
-                    <div class="form-group">
-                        <textarea class="input-message" id="connect" name="connect"
-                            placeholder="Hey I'd like to connect to..."></textarea>
-                        <img class="union" src="img/Union.png">
-                    </div>
-                    <div class="form-group">
-                        <button class="btn-2" type="button"><span class="text">Get in touch</span></button>
-                    </div>
-                </form>
+              <div id="one"></div>
+              <h4 class="header-3">Languages</h4>
+              <ul class="image-container-1">
+                <li><img src="img/js.png" alt="javascript" /></li>
+                <li><img src="img/rub.png" alt="ruby" /></li>
+                <li><img src="img/ht.png" alt="html" /></li>
+                <li><img src="img/cs.png" alt="css" /></li>
+              </ul>
             </div>
+          </div>
+          <div class="grid-item-1">
             <div>
-                <h1 class="email">ngachaian@gmail.com</h1>
+              <div id="two"></div>
+              <h4 class="header-3">Frameworks</h4>
+              <ul class="image-container-1">
+                <li><img src="img/react.png" alt="React" /></li>
+                <li><img src="img/rails.png" alt="ruby" /></li>
+                <li><img src="img/rspec.png" alt="Rspec" /></li>
+                <li><img src="img/capy.png" alt="Capybara" /></li>
+                <li><img src="img/selenium.png" alt="Selenium" /></li>
+              </ul>
             </div>
+          </div>
+          <div class="grid-item-1">
             <div>
-                <ul class="images-1">
-                    <li><a href=""><img src="img/twitter.png" alt="Twitter"></a></li>
-                    <li> <a href=""><img src="img/Linkedin icon.png" alt="Linkedin"></a></li>
-                    <li><a href=""> <img src="img/medium.png" alt="Medium"></a></li>
-                    <li><a href=""><img src="img/github.png" alt="Github"></a></li>
-                    <li><a href=""><img src="img/angellist.png" alt="Angellist"></a></li>
-                </ul>
+              <div id="three"></div>
+              <h4 class="header-3">Skills</h4>
+              <ul class="image-container-1">
+                <li><img src="img/database.png" alt="database" /></li>
+                <li><img src="img/git.png" alt="git" /></li>
+                <li><img src="img/cli.png" alt="cli" /></li>
+                <li><img src="img/web.png" alt="web" /></li>
+                <li><img src="img/api.png" alt="api" /></li>
+              </ul>
             </div>
-            <div id="four"></div>
+          </div>
+          <div>
+            <h1 class="header-4">
```

Comment:
- [ ] Kindly use the h2 ..... hx tag, because is advisable to h1 once if you want that google (and other search engines) to know about a specific topic.